geo-location.html

```diff
@@ -1,217 +1,174 @@
-<!-- Copyright Eric Bidelman <ebidel@gmail.com> -->
-
-<link rel="import" href="../polymer/polymer.html">
-
-<!-- TODO: decide if we need to cache results across instances. -->
-
-<!--
-Provides the current location using the [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation).
-
-<b>Example - basic</b>
-
-    <geo-location></geo-location>
-
+<link rel='import' href='../bower_components/polymer/polymer-element.html'>
```

Comment:
- Remove "bower_components' from the import.
- Remove the commented out elements.